db/migrate/20150612135546_create_host_status.rb

```diff
@@ -11,7 +11,29 @@ def up
     add_column :hosts, :global_status, :integer, :default => 0, :null => false
 
     Host.includes(:host_statuses, :last_report_object).find_each do |host|
```

Comment:
The `#find_each` method uses `#find_in_batches` with a batch size of 1000 (or as specified by the :batch_size option): http://apidock.com/rails/ActiveRecord/Batches/find_each